_dev/src/components/commons/banner-ads.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <b-card
+    class="border-0"
+    :class="{ 'shadow-sm mb-3' : size !== 'xs' }"
+  >
+    <button
+      v-if="isBanner"
+      type="button"
+      class="close"
+      aria-label="Close"
+      @click="closeBanner"
+    >
+      <span aria-hidden="true">&times;</span>
+    </button>
+    <div class="d-flex align-items-center">
+      <img
+        :src="require(`@/assets/images/${pngBanner}`)"
+        alt="banner-price"
+        :height="size === 'xs' ? 90 : 140"
+        :width="size === 'xs' ? 90 : 140"
+        class="mr-4"
```

Comment:
If you decide to update the responsivity, you have to adapt the margin here.  
It's just a suggestion, please do test it before merging anything.

```suggestion
        class="mr-md-4 mb-3 mb-md-0"
```

Reply:
Nothing changes when I use these classes

Reply:
`mr-md-4`: margin-left 24px starting from breakpoint md and up 
`mr-md-4`: margin-left 24px starting from breakpoint md and up

Reply:
`mr-md-4`: margin-left 24px starting from breakpoint md and up  
`mb-3`: margin-bottom 16px   
`mb-md-0`: margin-bottom 0px starting from breakpoint md and up   

It's if you decided to go with `flex-column flex-md-row` on the parent.